distribution/changelog.txt

```diff
@@ -2,6 +2,8 @@
 ------------------------------------------------------------------------
 - Improved: [#21424] Extra viewports can now rotate independently from the main viewport.
 - Improved: [#21599] Currency signs now use non-breaking spaces.
+- Improved: [#21561, #21631] Enable more features in Android build (plugins,
+networking, multiplayer, audio formats).
```

Comment:
This should be one line, not two.

Reply:
Oh, I don't know why it hard-wrapped, I looked at this line and was sure it was soft-wrapoed in my editor. Doh.